autogpt_platform/autogpt_libs/autogpt_libs/supabase_integration_credentials_store/types.py

```diff
@@ -8,6 +8,7 @@ class _BaseCredentials(BaseModel):
     id: str = Field(default_factory=lambda: str(uuid4()))
     provider: str
     title: Optional[str]
+    hidden: Optional[bool] = False
```

Comment:
1. "Hidden" isn't the right name for this, since it doesn't generally hide the item except on a specific screen.
2. Putting this here will also add and persist it to the DB on all new credentials created. This isn't necessary and should be avoided if possible.

Reply:
I've removed `hidden`, hardcoded hidden ids on the frontend and left a `TODO` for now.